week-2/G-conditionals-2/exercise-3.js

```diff
@@ -8,10 +8,32 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {
+/*function calculateGrade(mark) {
+  if (mark >= 80) {
+    return "A";
+  } else if (mark < 80 && mark > 60) {
+    return "B";
+  } else if (mark <= 60 && mark > 50) {
+    return "C";
+  } else {
+    return "F";
+  }
+}*/
 
+function calculateGrade(mark) {
+  if (mark >= 80) {
+    return "A";
+  } else {
+    if (mark > 60) {
+      return "B";
+    }
+    if (mark > 50) {
```

Comment:
I believe this should be >= 50, although I'm not 100% sure based on the instructions.